packages/assets/assets/error.svg

```diff
@@ -1,12 +1,12 @@
-<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="100%" height="100%" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
     <g clip-path="url(#clip0_3636_22875)">
         <path
             d="M7.99967 5.33203V7.9987M7.99967 10.6654H8.00634M14.6663 7.9987C14.6663 11.6806 11.6816 14.6654 7.99967 14.6654C4.31778 14.6654 1.33301 11.6806 1.33301 7.9987C1.33301 4.3168 4.31778 1.33203 7.99967 1.33203C11.6816 1.33203 14.6663 4.3168 14.6663 7.9987Z"
             stroke="#FFE5E5" stroke-width="1.33333" stroke-linecap="round" stroke-linejoin="round" />
```

Comment:
Let's change the stroke color to `#0D0D0D`. This way it will be clear when we view it in the IDE. 

> [!NOTE]
> The color in the design should not be changed. 

```suggestion
            stroke="#0D0D0D" stroke-width="1.33333" stroke-linecap="round" stroke-linejoin="round" />
```